x/tokenomics/keeper/token_logic_modules.go

```diff
@@ -468,6 +477,95 @@ func (k Keeper) TokenLogicModuleGlobalMint(
 	return nil
 }
 
+// TokenLogicModuleGlobalMintReimbursementRequest processes the business logic
+// for the GlobalMintReimbursementRequest TLM.
+func (k Keeper) TokenLogicModuleGlobalMintReimbursementRequest(
+	ctx context.Context,
+	service *sharedtypes.Service,
+	sessionHeader *sessiontypes.SessionHeader,
+	application *apptypes.Application,
+	supplier *sharedtypes.Supplier,
+	actualSettlementCoin cosmostypes.Coin,
+	relayMiningDifficulty *servicetypes.RelayMiningDifficulty,
+) error {
+	logger := k.Logger().With("method", "TokenLogicModuleGlobalMintReimbursementRequest")
+
+	// Do not process the reimbursement request if there is no global inflation.
+	if MintPerClaimedTokenGlobalInflation == 0 {
+		logger.Warn("global inflation is set to zero. Skipping Global Mint Reimbursement Request TLM.")
+		return nil
+	}
+
+	// Determine how much new uPOKT to mint based on global inflation
+	newMintCoin, _ := calculateGlobalPerClaimMintInflationFromSettlementAmount(actualSettlementCoin)
+	if newMintCoin.Amount.Int64() == 0 {
+		return tokenomicstypes.ErrTokenomicsMintAmountZero
+	}
+
+	// Update the application's on-chain stake
+	newAppStake, err := application.Stake.SafeSub(newMintCoin)
```

Comment:
I'm curious to understand if/how this interacts with the application min. stake and/or `maxClaimableAmount`. :thinking: 

_TL;DR, it seems to me like min. stake SHOULD be collateral for app overservicing AND GMRR. Is this that unnecessary? Do we need to update `MaxClaimableAmount`?_

`MaxClaimableAmount` prevents `actualSettlementCoin` from exceeding `application stake - min. stake` (once we add the unsettled sessions coefficient :sweat_smile:) but it seems like with GMRR we would also need to also account for the potential magnitude of `actualSettlementCoin + newMintCoin`. Wouldn't min. stake actually need to cover `actualSettlementCoin + newMintCoin` for **each claim** in **each session** between the current and currently settling sessions?

Since `newMintCoin` is a function of `actualSettlementCoin`, it seems like we should be able to consider it in the same scope as `maxClaimableAmount` if that's the way to go here.